src/platform/unix/mod.rs

```diff
@@ -1096,7 +1099,7 @@ fn create_shmem(name: CString, length: usize) -> c_int {
 #[cfg(all(feature = "memfd", target_os = "linux"))]
 fn create_shmem(name: CString, length: usize) -> c_int {
     unsafe {
-        let fd = memfd_create(name.as_ptr(), libc::MFD_CLOEXEC as usize);
+        let fd = libc::memfd_create(name.as_ptr(), libc::MFD_CLOEXEC);
```

Comment:
Oh hmm, glibc support was only added in 2.27. Not sure what the glibc policy of ipc-channel is.

Reply:
FWIW I think the old call was incorrect, since it passed in a usize but the syscall is a cint: https://github.com/torvalds/linux/blob/4d939780b70592e0f4bc6c397e52e518f8fb7916/mm/memfd.c#L332

Reply:
Looks like glibc 2.27 came out in 2018: https://sourceware.org/glibc/wiki/Release/2.27.